Automattic/wp-calypso
# WooCommerce: Add fields for simple product type

This adds the other product configuration fields that should be present when a product is configured as a 'simple' product (non-variable).

This includes price input, weight, and stock settings.

There are two `TODO`s in this file: The currency, currency position, dimensions unit, and weight unit all need to be passed in to the product form still (they are currently hardcoded). I have opened #14185 for the product form as a whole, so we make sure to come back and handle these.

cc @kellychoffman @jameskoster as this contains UI elements.
Closes #13664, #13665, #13664.

To test:
* Go to `http://calypso.localhost:3000/store/products/:site/add`.
* Toggle to variations and make sure those cards disappear and the variations UI is displayed instead. 'Simple' fields are dropped from state (like how we drop variation attributes when switching to simple). Toggle the variations switch again and make sure you can see the simple fields again.
* Configure the following various product settings and make sure their values update: Price, dimensions, weight, stock (checkbox, quantity), and backorders.

<img width="713" alt="screen shot 2017-05-17 at 10 20 41 am" src="https://cloud.githubusercontent.com/assets/689165/26167487/c012cf9e-3aeb-11e7-993b-680fe55eef9a.png">
